src/NetscriptFunctions.ts

```diff
@@ -593,123 +593,136 @@ export const ns: InternalAPI<NSFull> = {
-    if (!(server instanceof Server)) {
-      helpers.log(ctx, () => "Cannot be executed on this server.");
-      return false;
-    }
-    if (!Player.hasProgram(CompletedProgramName.relaySmtp)) {
-      throw helpers.makeRuntimeErrorMsg(ctx, "You do not have the relaySMTP.exe program!");
-    }
-    if (!server.smtpPortOpen) {
-      helpers.log(ctx, () => `Executed relaySMTP.exe on '${server.hostname}' to open SMTP port (25).`);
-      server.smtpPortOpen = true;
-      ++server.openPortCount;
-    } else {
-      helpers.log(ctx, () => `SMTP Port (25) already opened on '${server.hostname}'.`);
-    }
-    return true;
-  },
-  httpworm: (ctx) => (_hostname) => {
-    const hostname = helpers.string(ctx, "hostname", _hostname);
-    const server = helpers.getServer(ctx, hostname);
-    if (!(server instanceof Server)) {
-      helpers.log(ctx, () => "Cannot be executed on this server.");
-      return false;
-    }
-    if (!Player.hasProgram(CompletedProgramName.httpWorm)) {
-      throw helpers.makeRuntimeErrorMsg(ctx, "You do not have the HTTPWorm.exe program!");
-    }
-    if (!server.httpPortOpen) {
-      helpers.log(ctx, () => `Executed HTTPWorm.exe on '${server.hostname}' to open HTTP port (80).`);
-      server.httpPortOpen = true;
-      ++server.openPortCount;
-    } else {
-      helpers.log(ctx, () => `HTTP Port (80) already opened on '${server.hostname}'.`);
-    }
-    return true;
-  },
-  sqlinject: (ctx) => (_hostname) => {
-    const hostname = helpers.string(ctx, "hostname", _hostname);
-    const server = helpers.getServer(ctx, hostname);
-    if (!(server instanceof Server)) {
-      helpers.log(ctx, () => "Cannot be executed on this server.");
-      return false;
-    }
-    if (!Player.hasProgram(CompletedProgramName.sqlInject)) {
-      throw helpers.makeRuntimeErrorMsg(ctx, "You do not have the SQLInject.exe program!");
-    }
-    if (!server.sqlPortOpen) {
-      helpers.log(ctx, () => `Executed SQLInject.exe on '${server.hostname}' to open SQL port (1433).`);
-      server.sqlPortOpen = true;
-      ++server.openPortCount;
-    } else {
-      helpers.log(ctx, () => `SQL Port (1433) already opened on '${server.hostname}'.`);
-    }
-    return true;
-  },
+      
+    },
+    brutessh: (ctx) => (_hostname) => {
+      const hostname = helpers.string(ctx, "hostname", _hostname);
+      const server = helpers.getServer(ctx, hostname);
+      if (!(server instanceof Server)) {
+        helpers.log(ctx, () => "Cannot be executed on this server.");
+        return false;
+      }
+      if (!Player.hasProgram(CompletedProgramName.bruteSsh)) {
+        helpers.log(ctx,() => "You do not have the BruteSSH.exe program!");
+        return false;
+      }
+      if (!server.sshPortOpen) {
+        helpers.log(ctx, () => `Executed BruteSSH.exe on '${server.hostname}' to open SSH port (22).`);
+        server.sshPortOpen = true;
+        ++server.openPortCount;
+        return true;
```

Comment:
Is there a reason that you moved the `return true;` inside the if/else block? From visibility point of view I liked the `return true;` as the 'closer' of the function